ember-app/app/controllers/issue/create.js

```diff
@@ -21,19 +22,52 @@ var IssuesCreateController = Ember.ObjectController.extend({
     return this.get("model.title");
   }.property("model.title"),
   mentions: function(){
-    return _.uniq(_.compact(this.get('controllers.application.model.board.assignees')), function(i){
+    return _.uniq(_.compact(this.get('selectedBoard.assignees')), function(i){
       return i.login;
     });
-  }.property('controllers.application.model.board.assignees'),
+  }.property('selectedBoard','selectedBoard.assignees'),
   order: {},
   createIssue: function(order){
-    if (this.get("processing")){ return; }
+    if (this.get("processing")){ 
+      return; 
+    }
     var controller = this;
     this.set("processing",true);
-    this.get("model").saveNew(order).then(function(issue){
+    this.get("model").save(order).then(function(issue){
+       if(controller.get("issueIsLinked")){
+         controller.colorIssue(issue);
+         controller.send("assignRepo", controller.get("controllers.application.model.board"));
```

Comment:
I left it there so the controller can reset the model properties for labels, assignee and milestone